Remotion/Data/DomainObjects.PerformanceTests/Data.DomainObjects.PerformanceTests.csproj

```diff
@@ -29,5 +30,8 @@
     <ProjectReference Include="..\DomainObjects\Data.DomainObjects.csproj" />
     <ProjectReference Include="..\Core\Data.Core.csproj" />
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Logging" />
```

Comment:
isn't this a transitive dependency?

Reply:
No because our `Core.Logging.Leg4Net` only depends on the logging abstraction `Microsoft.Extension.Logging.Abstractions` but to create `LoggerFactory` we need the non-abstraction type.